### PR TITLE
Address #270 basic component for search  settings implemented

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
Addresses #270 
Changes: created search component and added some basic html.Later extensions to search settings are listed as sub issues at #270.
Made this pull so that work regarding the implementation of instant search could be started.
Demo Link: https://susper-pr-451.herokuapp.com/

Screenshots for the change: 

image

### CODE REVIEW

File: src/app/searchsettings/searchsettings.component.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <div class="leftPane col-md-2">
+        <ul>
+          <li ><a class="active" href="#">Search Results</a></li>
+          <li><a href="#">Languages</a></li>
+          <li><a href="#">Help</a></li>
+        </ul>
+    </div>
+    <div class="col-md-9">
+        <div>
+          <h4><strong>SafeSearch Filters</strong>/</h4>
```
Comment (Marauderer97) [inline src/app/searchsettings/searchsettings.component.html]: Please remove the extra /

File: src/app/searchsettings/searchsettings.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-searchsettings',
+  templateUrl: './searchsettings.component.html',
+  styleUrls: ['./searchsettings.component.css']
+})
+export class SearchsettingsComponent implements OnInit {
+  noofresults = 10;
```
Comment (sandeepm96) [inline src/app/searchsettings/searchsettings.component.ts]: Could it be a more meaningful identifier like resultCount or maxResults?

**Comment by Marauderer97** [comment]
@nikhilrayaprolu Great work! Two things though
1. 
image
I don't think more than one radio button should be allowed to be selected

2.
The radio buttons/ check boxes are slightly out of line with their labels, and also have no space from them.

**Comment by mariobehling** [comment]
@Marauderer97 You stated that this feature let's users select 3 buttons at the same time. So apparently this PR was not ready. 

Also: Only merge, when two other contributors have approved a PR, please.

**Pull request merged**